plugins/database/CMakeLists.txt

```diff
@@ -64,8 +64,8 @@ foreach(plugin IN LISTS IRODS_DATABASE_PLUGINS)
     ${plugin_target}
     PRIVATE
     irods_common
-    irods_server
     irods_plugin_dependencies
+    irods_server
```

Comment:
What motivated this change?

Reply:
I had made some changes in this stanza before but backed them out and didn't back this out. Eh, alphabetized. Can revert

Reply:
It's likely fine. I know there are situations where link order matters. I'm not so sure it matters here though.

I'd recommend to leave it as it was originally to avoid potential issues in case this does depend on link order.

Perhaps CMake frees us from that situation though.